rotkehlchen/types.py

```diff
@@ -881,6 +881,11 @@ def deserialize_from_db(cls, entry: tuple[int, str, str, str, int, int]) -> 'Use
         )
 
 
+class FValWithTolerance(NamedTuple):
+    value: FVal
+    tolerance: FVal = FVal('0')
```

Comment:
```suggestion
    tolerance: FVal = ZERO
```

Reply:
Moved some things around, to solve circular dependency issues for importing ZERO here.